File: Basic/Functions/223205E.c
#include <stdio.h>
#define intll long long

void swap(int *a, int *b) {
	int temp = a;
	a = b;
	b = temp;
}

void solution() {

	int a, b;
	scanf("%d %d", &a, &b);

	swap(&a, &b);
	printf("%d %d\n", a, b);
}

int main() {
	char TEST_CASE = 0;
	int t = 1;
	if (TEST_CASE == 1) {
		scanf("%d", &t);
	}

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Functions/223205E.go
package main

import (
	"fmt"
)

func solution() {

	var a, b int
	fmt.Scan(&a, &b)

	a, b = b, a

	fmt.Println(a, b)
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
